new-log-viewer/src/App.tsx

```diff
@@ -1,7 +1,90 @@
+import {
+    CssVarsProvider,
+    extendTheme,
+} from "@mui/joy/styles";
+
+import KeyboardArrowDown from "@mui/icons-material/KeyboardArrowDown";
+
 import Layout from "./components/Layout";
 import StateContextProvider from "./contexts/StateContextProvider";
 import UrlContextProvider from "./contexts/UrlContextProvider";
+import {CONFIG_KEY} from "./typings/config";
+import {CONFIG_DEFAULT} from "./utils/config";
+
 
+const monacoTheme = extendTheme({
+    colorSchemes: {
+        light: {
+            palette: {
+                success: {
+                    solidBg: "#007acc",
+                    solidHoverBg: "#0062a3",
+                    solidActiveBg: "#0062a3",
+                },
+                neutral: {
+                    solidBg: "#5f6a79",
+                    solidHoverBg: "#4c5561",
+                    solidActiveBg: "#4c5561",
+                },
+                focusVisible: "#0090f1",
+            },
+        },
+        dark: {
+            palette: {
+                success: {
+                    solidBg: "#0e639c",
+                    solidHoverBg: "#1177bb",
+                    solidActiveBg: "#1177bb",
+                },
+                neutral: {
+                    solidBg: "#313131",
+                    solidHoverBg: "#45494e",
+                    solidActiveBg: "#45494e",
+                },
+                focusVisible: "#007fd4",
+            },
+        },
+    },
+    focus: {
+        default: {
+            outlineWidth: "3px",
+        },
+    },
+    fontFamily: {
+        body: "-apple-system, system-ui, BlinkMacSystemFont, 'Segoe UI', Roboto, 'Helvetica Neue', Arial, sans-serif",
+    },
+    components: {
+        JoyButton: {
+            styleOverrides: {
+                root: ({ownerState}) => ({
+                    borderRadius: "2px",
+                    ...("md" === ownerState.size && {
```

Comment:
Is there any reason to have the customizations be `md` size specific?

Reply:
Nope. I think this is kept in our offline discussion for possible customization. Shall we remove it?

Reply:
Sure, let's remove it. If we do not need to check the size, we can simply provide the overrides in an object like 
```
root: {
    borderRadius: "2px",
    ...
},
```
right?